src/parsers/eventParser.js

```diff
@@ -135,6 +175,55 @@ export const parseSqsEvent = event => {
   return new_sqs_event;
 };
 
+export const parseS3Event = event => {
+  const new_s3_event = {};
```

Comment:
I don't know why, but in this page every parser is written in snake-case and not camel-case